docs/developer/release-team.md

```diff
@@ -0,0 +1,70 @@
+
+## Duration of Term
+
+Each release team term will last for a quarter, to align with one minor release cycle. A minor release cycle starts right after a minor release and concludes with the release of the next minor release.
+
+As noted above, making changes to  the CAPI release cadence is out of scope for this initial release team process. 
+
+## Specific Responsibilities
+
+- Release patch releases monthly or more often as needed, so users will get fixes & updated dependencies with CVE fix on a predictable cadence
+- Release quarterly (to be revised, see future work) minor releases, so users will get new features on a predictable cadence
+- Monitor CI signal, so a release can be cut at any time, and add CI signal for each new release branch
+- Maintain and improve user facing documentation about releases, release policy and calendar
+- Update the CAPI Netlify book certificates and DNS
+- Update the clusterctl Homebrew formula
+- Create and maintain the GitHub release milestone
+- Maintain and improve and release automation, tooling & related developer docs
+- Track tasks needed to add support for new Kubernetes versions
+
+
+## Team Roles
+
+- **Release Lead**: responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring that a retrospective happens. The lead is also responsible for ensuring a successor is selected and trained for future release cycles.
+- **Communications/Docs/Release Notes Manager**: Responsible for communicating key dates to the community, improving release process documentation, and polishing release notes. Also responsible for ensuring the user-facing Netlify book and provider upgrade documentation are up to date.
+- **CI Signal/Bug Triage/Automation Manager**: Assumes the responsability of the quality gate for the release and makes sures blocking issues and bugs are triaged and dealt with in a timely fashion. Helps improve release automation and tools.
+- **Shadow**: Any Release Team member may select one or more mentees to shadow the release process in order to help fulfill future Release Team staffing requirements and continue to grow the CAPI community in general.
+
+## Team Selection
+
+To start, the release team will be assembled by the release team lead based on volunteers. A call for volunteers can be made through the usual communication channels (office hours, Slack, mailing list, etc.). In the future, we may consider introducing an application process similar to the Kubernetes release team application process. 
+
+### Selection Criteria
+
+When assembling a release team, the release team lead should look for volunteers who:
+
+- Can commit to the amount of time required across the release cycle
```

Comment:
Given the term is a quarter I wonder if we shouldn't have a backup plan for each of the roles. Example: Release lead has to step away for a while (life or work related) who takes over? does the lead find a replacement, do the Comms or CI person take over?

Reply:
I think we'd want to have someone take over at that point, maybe the shadow can take over? We can also deal with this on a case by case basis to start and formalize the process as we learn from the first iterations

Reply:
We’ve had a few instances of this with the Kubernetes release team. Shadows help, having folks that can step back in alsonis very helpful for these kind of scenarios. Definitely plan for it to happen, lots of things can happen in a quarter.

Reply:
This is great to think about and plan for. We’ve had a few instances of this happening and it’s worked pretty well to have the shadows step up. One thing that can help this is to make good runbook/handbooks for what needs to happen and when it needs to happen (and update that as you find better ways of working).